src/chart/generateCategoricalChart.tsx

```diff
@@ -1869,7 +1869,7 @@ export const generateCategoricalChart = ({
 
       if (tooltipEventType !== 'axis' && tooltipItem && tooltipItem.props.trigger === 'click') {
         itemEvents = {
-          onClick: combineEventHandlers(this.handleItemMouseEnter, element.props.onClick),
+          // onClick: combineEventHandlers(this.handleItemMouseEnter, element.props.onClick),
```

Comment:
Oh wait I didn't mean to commit this

Reply:
Tests passed though - if anything this shows a lack of coverage 😢

Reply:
Oops